core/trino-main/src/test/java/io/trino/execution/TestCreateMaterializedViewTask.java

```diff
@@ -181,14 +189,39 @@ public void testCreateMaterializedViewWithInvalidProperty()
                 ImmutableList.of(new Property(new Identifier("baz"), new StringLiteral("abc"))),
                 Optional.empty());
 
-        assertTrinoExceptionThrownBy(() -> getFutureValue(new CreateMaterializedViewTask(metadata, new AllowAllAccessControl(), parser, analyzerFactory)
+        assertTrinoExceptionThrownBy(() -> getFutureValue(new CreateMaterializedViewTask(metadata, new AllowAllAccessControl(), parser, analyzerFactory, new FeaturesConfig())
                 .execute(statement, queryStateMachine, ImmutableList.of(), WarningCollector.NOOP)))
                 .hasErrorCode(INVALID_MATERIALIZED_VIEW_PROPERTY)
                 .hasMessage("Catalog 'catalog' does not support materialized view property 'baz'");
 
         assertEquals(metadata.getCreateMaterializedViewCallCount(), 0);
     }
 
+    @Test
+    public void testCreateMaterializedViewWithDefaultProperties()
```

Comment:
nit: separate commit. How it's related to `ALTER MV` commit?

Reply:
This change is not in the `ALTER MV` commit, i.e. it is not in the commit "Support setting mv properties in core engine". It is in the commit "Support setting a property to DEFAULT in various statements" (and `CREATE MATERIALIZED VIEW` is one of those "various statements" (the list of "various statements" is in the body of the commit message))